src/OrchardCore.Modules/OrchardCore.GitHub/AdminMenuGitHubLogin.cs

```diff
@@ -34,6 +36,24 @@ protected override ValueTask BuildAsync(NavigationBuilder builder)
                 )
             );
 
+            return ValueTask.CompletedTask;
+        }
+
+        builder
+            .Add(S["Settings"], settings => settings
+                .Add(S["Security"], S["Security"].PrefixPosition(), security => security
+                    .Add(S["Authentication"], S["Authentication"].PrefixPosition(), authentication => authentication
```

Comment:
Rather External Authentication, for the others too.

Reply:
I would rather keep it Authentication. Why would we call it external authentication where is this the other authentication item we have in the menu. The other thing have is Access control

Reply:
The only items under this after enabling every feature are external authentication settings.

Reply:
Authentication would shorter with same meaning. Also if we ever need to add authentication settings related to local authentication that would go under an authentication otherwise we could end up with Authentication and External Authentication.

Reply:
Authentication is also about local authentication, the settings of which (related to login and registration) are not under this menu but a level up.

And yes, we could totally have a differentiation here, with a Local Authentication and External Authentication menu (or just one Authentication, but then having all authentication-related things under it).

Additionally, now we have Access Control, Security, and Authentication, which is confusing.

Reply:
**Access Control** is for managing access. Security and Authentication items under the settings menu are are different things.
Security are security related settings for the app itself like header, CORS, https..

If you think we should call it External Authentication, then maybe we should also move the local authentication under "Authentication" so we have two sub menus.

Reply:
Looking more at the current setup, I think Registration, User Login, and Change Email are great under "Security". External Authentication providers are under "Authentication" make sense to me without the word "External". If you want to suggest a new setup, please share a bullet bases structure so the picture becomes clear. The following looks great to me.

![image](https://github.com/user-attachments/assets/d2ce179b-a1fe-4388-bdec-29ed8c29618a)

Reply:
I also think that we don't need to have an extra "external" authentication. External vs. local is still all "Authentication" and easy to find there.